src/contributor-guide/contributing.md

```diff
@@ -273,6 +273,25 @@ The command merges the listed related pull requests (`xxx`, `yyy`, `zzz`) into t
 -  [Community Maintainers](https://github.com/magento/magento2/wiki/Community-Maintainers)
 -  Magento EngCom Team
 
+### Import source code to specific repository {#import}
+
+The import command provides the ability to copy a contributor's code or pull request into an internal fork.  The internal team can then proceed with additional fixes or delivery.
+
+**Command:** To import code or a pull request, a member of the Magento team controlling the pull request enters the following command:
+
+```text
+@magento import {code|pr} to {organizationName}/{repositoryName}
+```
+
```

Comment:
I find it very helpful to have samples of these types of commands. Can you please provide a sample of each?
And I am not clear on when you would pull in a contributors code without a PR? Is this the same as "Update Branch"?

Reply:
@dobooth examples are added

Reply:
Good improvement. Thanks!